lms/djangoapps/verify_student/models.py

```diff
@@ -1108,3 +1109,24 @@ def deadline_for_course(cls, course_key):
             return deadline.deadline
         except cls.DoesNotExist:
             return None
+
+
+class SSPVerificationRetryConfig(ConfigurationModel):
+    """
+        SSPVerificationRetryConfig used to inject arguments
+        to retry_failed_photo_verifications management command
+    """
+
+    class Meta(object):
+        app_label = 'verify_student'
+        verbose_name = 'sspv retry student argument'
+
+    arguments = models.TextField(
+        blank=True,
+        help_text='Useful for manually running a Jenkins job. Specify like "Verification id,  \
```

Comment:
Can we rephrase this line `Specify like "Verification id, Verification id`. Its seems consuming. Would be better if you can add example as well

Reply:
Done